django/urls/resolvers.py

```diff
@@ -297,14 +295,14 @@ def _route_to_regex(route, is_endpoint=False):
 
 
 class RoutePattern(CheckURLMixin):
-    regex = LocaleRegexDescriptor("_route")
+    regex = LocaleRegexDescriptor("_regex")
 
     def __init__(self, route, name=None, is_endpoint=False):
         self._route = route
         self._regex_dict = {}
         self._is_endpoint = is_endpoint
         self.name = name
-        self.converters = _route_to_regex(str(route), is_endpoint)[1]
+        self._regex, self.converters = _route_to_regex(str(route), is_endpoint)
```

Comment:
Interestingly, this double execution of `_route_to_regex` dates back to its addition in df41b5a05d4e00e80e73afe629072e37873e767a . It must have been missed in that initial mega PR!